### PR TITLE
ADM-362 T8 - [frontend] [Metrics] Real done part II

### PR DESCRIPTION
**Pull request by spadio9587**
## Summary
fix the realDone part showing logic

Given users haven't select anything for Done attribute or select except "Done" candidate
When users select "Done" candidate for other attributes beside Done attribute
Then the Real Done section will disappear
...

## Before

_Description_

Before

## After

_Description_

After

## Note

_Null_

### CODE REVIEW

File: frontend/src/components/Metrics/MetricsStep/RealDone/index.tsx [hunk 1]
```diff
@@ -63,14 +59,21 @@ export const RealDone = ({ options, title, label }: realDoneProps) => {
...
+            return (
+              <MenuItem key={column} value={column}>
+                <Checkbox checked={isChecked} />
+                <ListItemText primary={column} />
+              </MenuItem>
+            )
+          })}
         </Select>
-        <FormHelperText>{isEmptyRealDoneData ? errorHelperText() : DEFAULT_HELPER_TEXT}</FormHelperText>
+        {isEmptyRealDoneData && (
```
Comment (guzhongren) [inline frontend/src/components/Metrics/MetricsStep/RealDone/index.tsx:72]: set default helper text here，ref to main branch

Comment (spadio9587) [reply frontend/src/components/Metrics/MetricsStep/RealDone/index.tsx:72]: got that, will fix that soon

**Review by mengna-tw — APPROVED** [review]

File: frontend/__tests__/src/components/Metrics/MetricsStep/MetricsStep.test.tsx [hunk 2]
```diff
@@ -27,7 +28,7 @@ const setup = () =>
   )
 
 describe('MetricsStep', () => {
-  it('should render Crews and RealDone components', () => {
+  it('should render Crews components', () => {
```
Comment (Teresasasa) [inline frontend/__tests__/src/components/Metrics/MetricsStep/MetricsStep.test.tsx:31]: It seems that not only crews components in the document

Comment (spadio9587) [reply frontend/__tests__/src/components/Metrics/MetricsStep/MetricsStep.test.tsx:31]: Got that, will double check and fix that later.

**Review by LanxinWang — APPROVED** [review]
LGTM

**Review by Teresasasa — APPROVED** [review]
LGTM

**Pull request merged**